apps/web/src/components/Onboarding/SetupPassword/SetupPassword.tsx

```diff
@@ -101,14 +120,16 @@ export const SetupPassword = ({ mode }: SetupPasswordProps) => {
           });
           break;
         }
+        default:
```

Comment:
please don't include the default as it breaks [switch-exhaustiveness-check](https://typescript-eslint.io/rules/switch-exhaustiveness-check/)